cases/aws_ebs_volume/README.rst

```diff
@@ -17,7 +17,7 @@ Unsupported attributes
 Notes
 ~~~~~
 
-``gp2`` and ``st2`` are valid values for ``type`` attribute. Disks with ``st2`` volume type must have ``size`` attribute value more then ``32G`` and be multiple of 8GiB. The volume size for ``gp2`` volume type varies from 8 GiB to 4 TiB. The volume size must be multiple of 8 GiB. For more information visit documentation `page <https://docs.cloud.croc.ru/en/services/instances_and_volumes/volumes.html>`_.
+``gp2``, ``io2`` and ``st2`` are valid values for ``type`` attribute. Disks with ``st2`` volume type must have ``size`` attribute value more then ``32G`` and be multiple of 8GiB. The volume size for ``gp2`` and ``io2`` volumes type varies from 8 GiB to 4 TiB. The volume size must be multiple of 8 GiB. The ``io2`` volumes support the ``iops`` option, it's necessary to define it for ``io2`` volume type in the range from ``100`` to ``50000``.  For more information visit documentation `page <https://docs.cloud.croc.ru/en/services/instances_and_volumes/volumes.html>`_.
```

Comment:
Not sure about necessary to define iops, but it could be define. If we don't define iops it will be default value and equal 400 iops.

Reply:
It will not be equal 400 iops after the returning back default behaviour. This task on test now.